internal/api/handlers/operations_handler.go

```diff
@@ -152,7 +152,7 @@ func sortOperationsByCreatedAt(operations []*operation.Operation, order string)
 }
 
 func extractSortingParameters(orderBy string) (string, error) {
-	sortingOrder := "asc"
+	sortingOrder := "desc"
```

Comment:
@caiordjesus @lyracampos We were using the wrong sorting order here, we want to show the latest operations first (decrescent order) if the sorting parameters aren't provided